.travis.yml

```diff
@@ -110,6 +96,17 @@ install:
         make --no-print-directory -s pkg-utils pkg-kmod || travis_terminate 1;
         sudo dpkg -i *.deb || travis_terminate 1;
       fi
+    # Return to libcstor code to complie zrepl which is responsible for uZFS feature.
+    # Save the current location to get back
+    - pushd .
+    - cd ..
+    - cd libcstor
+    # Go to zrepl directory to build zrepl
+    - cd cmd/zrepl
+    - make
+    - cd ../../
```

Comment:
as vitta said, no need to do cd ../../ just popd is sufficient.

Reply:
Updated PTAL